tests/services/test_orgs.py

```diff
@@ -33,51 +33,51 @@ def mock_get_all_empty_response(self, mocker):
         return Py42Response(response)
 
     def test_get_org_by_id_calls_get_with_uri_and_params(
```

Comment:
This test seems like it is not the right directory as well... This is testing a Service but it is in `tests/clients/`